### PR TITLE
Hw05

### PR DESCRIPTION
**Pull request by mkharkhu**
Ready to fight

### CODE REVIEW

File: hw05/src/Fruit.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    private double price;
+
+    public Fruit(FruitType type, int timeOfLife, LocalDate receiptDate, int price) {
+        this.type = type;
+        this.timeOfLife = timeOfLife;
+        this.receiptDate = receiptDate;
+        this.price = price;
+    }
+
+    boolean valid(LocalDate date){
```
Comment (SergeyPasko) [inline hw05/src/Fruit.java]: POJO it is not place for logic

File: hw05/src/Shop.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    List<Fruit> freshFruitsOfOneType(FruitType type, LocalDate date){
+        List<Fruit> listFreshAndType = new ArrayList<>();
+        for(Fruit item : list){
+            if(item.getType().equals(type) && item.valid(date))
+                listFreshAndType.add(item);
+        }
+        return listFreshAndType;
+    }
+
+    List<Fruit> modifiedPrice(FruitType type, int percent, int days){
```
Comment (SergeyPasko) [inline hw05/src/Shop.java]: Read homeTask. You must use many FruitTypes as param (List\<FruitType\> or FruitType[] or FruitType...)

File: hw05/src/Fruit.java (hunk 1 above)
Comment (mkharkhu) [reply hw05/src/Fruit.java]: what does it mean?

Comment (SergeyPasko) [reply hw05/src/Fruit.java]: POJO pattern

File: hw05/src/Shop.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    }
+
+    List<Fruit> modifiedPrice(int percent, int days, FruitType... type){
+        List<Fruit> listOfModifiedPrice = new ArrayList<>();
+
+        for(FruitType type1 : type){
+            for(Fruit item : list){
+                if(item.getType().equals(type1)){
+                    if(valid(item, LocalDate.now())){
+                        item.setPrice(item.getPrice() / 100 * (100 - percent));
```
Comment (SergeyPasko) [inline hw05/src/Shop.java]: =)

**Pull request merged**